wdl/transforms/draft3/src/test/scala/wdl/draft3/transforms/wdlom2wom/WdlFileToWomSpec.scala

```diff
@@ -167,4 +170,34 @@ class WdlFileToWomSpec extends FlatSpec with Matchers {
     task.meta should be (Map("author" -> "John Doe",
                              "email" -> "john.doe@yahoo.com"))
   }
+
+  // There is a scatter within a scatter
+  //
+  // scatter(a in indices) {
+  //   scatter(b in indices) {
+  //     Int x = a + b
+  //   }
+  // }
+  //
+  private def validateTwoLevelScatter(b : WomBundle) : Assertion = {
+    val wf = b.primaryCallable.get.asInstanceOf[WorkflowDefinition]
+    val graph = wf.innerGraph
+
+    // get the top scatter node
+    graph.scatters.size shouldBe(1)
+    val topScatter : ScatterNode = graph.scatters.toVector.head
+    val wfCalls = graph.allNodes.filterByType[WorkflowCallNode]
+
+    if (WomParseConfig.convertNestedScatterToSubworkflow) {
+      // There should be a call to a generated sub-workflow in the graph
+      wfCalls.size shouldBe(1)
```

Comment:
I'm not sure this test is really checking anything because there is currently no means to vary the config value